py/datastore_service.py

```diff
@@ -139,6 +173,38 @@ def do_GET(self):
   def do_POST(self):
     self.do(True)
 
+def initialize_db():
+  #try to connect forever...
+  credentials = (os.environ['POSTGRES_DB'], os.environ['POSTGRES_USER'], os.environ['POSTGRES_HOST'], os.environ['POSTGRES_PASSWORD'])
+  while True:
+    try:
+      sql_conn = psycopg2.connect("dbname='%s' user='%s' host='%s' password='%s'" % credentials)
+      break
+    except Exception as e:
```

Comment:
i think you should write `str(e)` to `stderr` like is done above. i wonder if maybe we dont need to try forever above since you're doing it here now.